wordpress-mobile/gutenberg-mobile
# Upgrade CI to Xcode 12

Fixes #2732 

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/26283

To test:
- Check if CI is green, including full e2e test suite
- Run e2e tests locally using Xcode 12:
    1. Make sure command line is using Xcode 12: `sudo xcode-select --switch /Applications/Xcode_12_path.app`
    1. `npm run test:e2e:ios:local`

PR submission checklist:
